packages/react-focus/src/components/FocusZone/FocusZone.tsx

```diff
@@ -768,7 +776,7 @@ export class FocusZone extends React.Component<IFocusZoneProps> implements IFocu
    * Walk up the dom try to find a focusable element.
    */
   private _tryInvokeClickForFocusable(target: HTMLElement): boolean {
-    if (target === this._root.current || !this.props.shouldRaiseClicks) {
+    if (target === this._root.current) {
```

Comment:
I strongly believe checking for `shouldRaiseClicks` should not be the responsibility of this function so I moved the check into the appropriate switch cases.